staff/marta-penya/precourse/calculator-2/style.css

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Im not sure if client would like the colours of your calculator! LOL

Reply:
¡You're right! I should check and change the CSS of this project